iceoryx_hoofs/include/iceoryx_hoofs/cxx/string.hpp

```diff
@@ -441,6 +443,7 @@ class string
     /// @brief clears the content of the string
     constexpr void clear() noexcept;
 
+    // AXIVION Next Construct AutosarC++19_03-A13.5.2 , AutosarC++19_03-A13.5.3: TODO
```

Comment:
We need to deal with the TODO. If we cannot define it as explicit (as it breaks other ocde) we need a justification.

Justification: used for interoperability with `std::string`